python/ray/dashboard/dashboard.py

```diff
@@ -925,4 +927,7 @@ def clean_trials(self, trial_details, job_name):
                    "error:\n{}".format(os.uname()[1], traceback_str))
         ray.utils.push_error_to_driver_through_redis(
             redis_client, ray_constants.DASHBOARD_DIED_ERROR, message)
-        raise e
+        if isinstance(e, OSError) and e.errno != errno.ENOENT:
```

Comment:
Hmm, this should be `e.errno == errno.ENOENT`, right?

Reply:
Oops, yes, thanks for the catch!

Reply:
Pushed.